packages/contracts-bedrock/scripts/deploy/AddGameType.s.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+// Forge
+import { Script } from "forge-std/Script.sol";
+
+// Scripts
+import { DeployUtils } from "scripts/libraries/DeployUtils.sol";
+
+// Interfaces
+import { OPContractsManager } from "src/L1/OPContractsManager.sol";
+import { ISystemConfig } from "interfaces/L1/ISystemConfig.sol";
+import { IProxyAdmin } from "interfaces/universal/IProxyAdmin.sol";
+import { IDelayedWETH } from "interfaces/dispute/IDelayedWETH.sol";
+import { IBigStepper } from "interfaces/dispute/IBigStepper.sol";
+import { GameType, Duration, Claim } from "src/dispute/lib/Types.sol";
+import { IFaultDisputeGame } from "interfaces/dispute/IFaultDisputeGame.sol";
+
+/// @title AddGameType
+contract AddGameType is Script {
+    struct Input {
+        // Address that will be used for the DummyCaller contract
+        address prank;
+        // OPCM contract address
+        OPContractsManager opcm;
+        // SystemConfig contract address
+        ISystemConfig systemConfig;
+        // ProxyAdmin contract address
+        IProxyAdmin proxyAdmin;
+        // DelayedWETH contract address (optional)
+        IDelayedWETH delayedWETH;
```

Comment:
Can you align these names with the standard contract names in op-chain-ops/addresses (i.e. add `Proxy` suffix to all three and change proxyAdmin --> opChainProxyAdmin`)? Same comment for the go input/output structs that interface with this script